package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build --modern",
-    "format": "yarn run eslint --fix --ext .js,.vue src,tests",
+    "format": "yarn run eslint --fix --ext .js,.vue src/**",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏why tests are excluded?

Reply:
It wasn't working with the comma as a separator so I wasn't sure it was possible to specify both directories in one command, but I've found that it works fine if you separate them with a space, fix has been pushed 🙏